api/validator.js

```diff
@@ -36,7 +36,11 @@ const SCHEMA_JOB_PERMUTATION = Joi.object()
         secrets: Job.secrets,
         settings: Job.settings,
         sourcePaths: Job.sourcePaths,
-        stages: Base.stages,
+        stage: Joi.object()
+            .keys({
+                name: Joi.string().regex(Regex.STAGE_NAME).optional()
+            })
+            .unknown(false),
```

Comment:
Having `stage` should be optional. If stage object exists, it must have `name`
```suggestion
        stage: Joi.object()
            .keys({
                name: Joi.string().regex(Regex.STAGE_NAME).required()
            })
            .optional()
            .unknown(false),
```

Reply:
was troubleshooting something for config-parser and forgot to change it back